orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/api/OidcAPI.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.di.orchestration.shared.api;
+
+import uk.gov.di.orchestration.shared.services.ConfigurationService;
+
+import java.net.URI;
+
+import static uk.gov.di.orchestration.shared.helpers.ConstructUriHelper.buildURI;
+
+public class OidcAPI {
```

Comment:
Nitpick, but it seems strange to capitalise one initialism but not the other

Reply:
My understanding was that the convention for Java was upper case for 2-3 letter acronyms and camel case elsewhere. I prefer camel case to be honest because what if you have a 2 and three letter acronym next to each other for instance - hard to read. Do you have a preference?

Reply:
Ah interesting. Only preference is for consistency